src/main/java/picard/analysis/TheoreticalSensitivityMetrics.java

```diff
@@ -0,0 +1,14 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+/**
+ * Created by fleharty on 6/30/17.
+ */
+public class TheoreticalSensitivityMetrics extends MetricBase {
+    /** Theoretical Het Sensitivity for log-odss of 6.2 **/
```

Comment:
By fixed metrics, I assume you mean the TheoreticalHet sensitivities for different log odds.

I had a discussion with @davidbenjamin about the log odds threshold, and I think we need to rethink the threshold we use for TheoreticalHetSensitivity.  This was just a nice way to include THS with a log odds threshold of 6.2.

I'll remove these two because we get THS with log odds of 3.0 in the other metrics file.  And we get TS with log odds of 6.2 for 0.5 allele fraction.